spec/integration/working_file_path_spec.rb

```diff
@@ -0,0 +1,298 @@
+require 'rails_helper'
+
+# MasterFile#working_file_path has been a source of problems since it was introduced in 6.4.3
+# This spec file is meant to thoroughly test it in order to find any remaining bugs and to
+# ensure the intended functionality doesn't get broken accidentally in the future.
+#
+# Need to test all of the possiblilites:
+# batch ingest with single file
+# batch ingest with single file skip transcoding
+# batch ingest with pre-transcoded derivatives
+# web upload
+# web upload skip transcoding
+# BE dropbox upload
```

Comment:
No clue what BE is, might be worth unrolling the acronym (back end?)

Reply:
Sure, I'll change that now.